.github/workflows/benchmark.yml

```diff
@@ -241,18 +241,19 @@ jobs:
 
     - name: Compare results
       run: |
+        THRESHOLD=250
         if [ ${RELEASE_TIME} -le ${HEAD_TIME} ]; then
           delta=$(( 1000 * ( ${HEAD_TIME} - ${RELEASE_TIME} ) / ${RELEASE_TIME} ))
           echo "delta=$delta"
-          if [ 100 -lt ${delta} ]; then
+          if [ $THRESHOLD -lt ${delta} ]; then
             echo "Significantly slowdown. Was ${RELEASE_TIME}, now ${HEAD_TIME}. Delta(%*10)=${delta}"
             exit 1
           fi
```

Comment:
Didn't you told using `re` is faster than using `regex`?
Then why the slow down threshold is increased?

Reply:
Thank you for explanation.